extra/sponge.importorder

```diff
@@ -0,0 +1,7 @@
+#Organize Import Order
```

Comment:
Yeah it's for Eclipse, and a useful reference to know what order imports should be.